src/internal/util/isPromise.ts

```diff
@@ -1,3 +1,8 @@
+/**
+ * Tests to see if the object is an ES2015 (ES6) Promise
+ * @see {@link https://www.ecma-international.org/ecma-262/6.0/#sec-promise-objects}
+ * @param value the object to test
+ */
 export function isPromise(value: any): value is PromiseLike<any> {
```

Comment:
I believe the return type on this function should be a `boolean`, though I see that this matches the code in for `isObservable` [here](https://github.com/ReactiveX/rxjs/blob/master/src/internal/util/isObservable.ts#L8). I could be totally missing something, but this seems incorrect... Could someone on the team confirm??

Reply:
No, current return type definition is desired.

Reply:
I also wasn't familiar with this, but had left it because isObservable also did not have the boolean return type -
Thanks for the clarification @kozlown and the docs find @okeefem2 !